src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -30,7 +30,7 @@ public X getValue() {
         double a = firstBound.getValue().getReal();
         double b = secondBound.getValue().getReal();
 
-        if (v >= Math.min(a, b) && v <= Math.max(a, b)) {
+        if (v >= Math.min(a, b) && v < Math.max(a, b)) {
             return DFFactory.get(1d);
```

Comment:
Not in the scope, but there are only 7 chars to modify with no impact at all(as I understand it). Almost a free refactoring: probably worth it :)